build.gradle

```diff
@@ -228,6 +228,10 @@ dependencies {
 
   implementation group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version:'0.0.4'
 
+  implementation group: 'org.springframework', name: 'spring-context-support', version: '5.2.2.RELEASE'
+
+  implementation group: 'com.sun.mail', name: 'jakarta.mail', version: '1.6.3'
+
```

Comment:
Unusual to our setup comparing with previous integrations of reports. Can you elaborate more around this choice?